composer.json

```diff
@@ -4,7 +4,8 @@
     "description": "Flow Form Framework preset for Fusion based Form rendering",
     "license": "MIT",
     "require": {
-        "neos/form": "^5.0"
+        "neos/form": "^5.0",
+        "neos/fusion": ">=7.3"
```

Comment:
why didn't you use semantic version constraints, e.g. (`^7.3`)?

Reply:
Yeah, than we need to change it with each Neos version.

Reply:
And that makes sense! Never use unlimited constraints!
```suggestion
        "neos/fusion": "^7.3 || ^8.3 || ^9.0"
```

Reply:
```suggestion
        "neos/fusion": "^7.3 || ^8.0 || ^9.0"
```

"^8.0" right?

Reply:
🙈 Yes, of course!